### PR TITLE
20171213 Docs interlis updated

### PR DESCRIPTION
**Pull request by sjib**
Various minor changes based on open issues:
#44 (closed now)
#33 
Change Link to zip with Demodata #49 (closed now)

### CODE REVIEW

File: en/user-guide/editing/editing.rst [hunk 1]
```diff
@@ -135,6 +135,12 @@ The description is for an additional cover.
 
 .. figure:: images/new_cover_connect_to_wws.jpg
  
+
+Multiedit
+---------
+If you want to edit attributes for multiple objects of the same class you can use the `mulitedit Tool of QGIS <https://docs.qgis.org/2.18/en/docs/user_manual/working_with_vector/attribute_table.html#multi-edit-fields>`_.
```
Comment (sjib) [inline en/user-guide/editing/editing.rst]: mulitedit -> multiedit

**Comment by sjib** [comment]
Can this be merged?

File: en/admin-guide/data-export/index.rst [hunk 2]
```diff
@@ -1,13 +1,204 @@
...
+
+a) Generate schema with data model
+You need the wanted INTERLIS model with all related models:
+
+- *Units.ili*
+- *Base.ili*
+- *SIA405_Base.ili*
+- *VSA_DSS_2015_2_d_20170602.ili* for Version 2015 –> corrected version relations 
+
+>>> java -jar ili2pg.jar --createEnumTxtCol --schemaimport --importTid --sqlEnableNull --createEnumTabs --createFk  --noSmartMapping --dbdatabase qgep --dbschema vsa_dss_2015_2_d_391 --dbusr postgres --dbpwd sjib  --log createschema_VSA_DSS_2015_2_d_391.log VSA_DSS_2015_2_d_20170602.ili
```
Comment (m-kuhn) [inline en/admin-guide/data-export/index.rst]: Should be formatted as code (`.. code::` I think)

File: en/admin-guide/data-import/index.rst [hunk 3]
```diff
@@ -1,14 +1,178 @@
...
+
+>>> rem bat and sql in the same directory
+set PATH=%PATH%;C:\Program Files\PostgreSQL\9.4\bin
+psql -U postgres -d vsa_dss_2015_2_d_304 -f 01_vsa_dss_2015_2_d_304_tid_generate.sql
+psql -U postgres -d vsa_dss_2015_2_d_304 -f 02_vsa_dss_2015_2_d_304_tid_lookup.sql
+pause
+
+Specify the path variable to postgresSQL on your system
+
+``-u`` sets the user
```
Comment (m-kuhn) [inline en/admin-guide/data-import/index.rst]: Capital or not (difference between example above and explanation here)

File: en/admin-guide/data-import/index.rst [hunk 4]
```diff
@@ -1,14 +1,178 @@
...
+ * If data is exported from DABASE or other systems it may be that the **reach**.trace has double point or that is going from **wastewater_node** to **wastewater_node** instead of **reach_point** to **reach_point**. Looking for a solution to automatically check after import and "clean" linestring.
+ 
+ 
+ 
+Possible problems
+-------------------
+
+* qgep schema of demodata has missing foreignkeys: Einleitstelle.fs_Gewaessersektor, Versickerungsanlage.fs_Grundwasserleiter, Abwasserknoten.fs_hydrgeomref –> check whether ok in empty schema ``qgep``.
+
+* Need to use ``ST_Force3D`` for geometries – because qgep model is now with 3D coordinates
```
Comment (m-kuhn) [inline en/admin-guide/data-import/index.rst]: remove the word "now"

**Comment by m-kuhn** [comment]
Just some minor comments from quickly looking through the proposed changes

File: en/admin-guide/data-export/index.rst (hunk 2 above)
Comment (sjib) [reply en/admin-guide/data-export/index.rst]: No - I think this looks fine - see preview

**Pull request merged**